Modules/_lzmamodule.c

```diff
@@ -1495,7 +1497,9 @@ _lzma__decode_filter_properties_impl(PyObject *module, lzma_vli filter_id,
 
     /* We use vanilla free() here instead of PyMem_Free() - filter.options was
        allocated by lzma_properties_decode() using the default allocator. */
-    free(filter.options);
+    if (filter.options) {
+        free(filter.options);
+    }
```

Comment:
It is not needed. `free()` accepts NULL.
```suggestion
    free(filter.options);
```